.github/workflows/gradle.yml

```diff
@@ -4,20 +4,21 @@ on: [ pull_request, push ]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Is there any reason why we shouldn't be using the latest distribution of `ubuntu`?

Reply:
The "latest" context is actually misleading as that redirects to re-using an older version of ubuntu which is 20.04 LTS instead.